packages/dd-trace/src/exporters/common/request.js

```diff
@@ -76,7 +81,7 @@ function request (data, options, keepAlive, callback) {
 
     dataArray.forEach(buffer => req.write(buffer))
 
-    req.setTimeout(timeout, req.abort)
+    req.setTimeout(timeout, () => req.destroy(new Error('Request timed out')))
```

Comment:
abort is deprecated, and with destroy we can throw an error that we can retry: https://nodejs.org/api/http.html#requestabort